src/composables/transactionList.ts

```diff
@@ -0,0 +1,324 @@
+import { useStore } from 'vuex';
+import { orderBy, uniqBy } from 'lodash-es';
+import {
+  computed, onMounted, ref, watch,
+} from 'vue';
+import { Encoded, Tag } from '@aeternity/aepp-sdk';
+import {
+  ITransaction,
+  ITransactionsState, ITransactionStateList,
+} from '../types';
+import { useMiddleware } from './middleware';
+import {
+  AEX9_TRANSFER_EVENT, fetchJson, getLocalStorageItem, setLocalStorageItem, TX_DIRECTION,
+} from '../popup/utils';
+import { useAccounts } from './accounts';
+import JsonBig from '../lib/json-big';
+import { useAeSdk } from './aeSdk';
+
+const LOCAL_STORAGE_KEY = ['transactionsState'];
+
+const isInitalized = ref(false);
+
+const generateEmptyTransactionState = (): ITransactionsState => ({
+  loaded: [],
+  nextPageUrl: '',
+  pending: {},
+  tipWithdrawnTransactions: [],
+});
+
+const transactions = ref<ITransactionStateList>(
+  getLocalStorageItem<ITransactionStateList>(LOCAL_STORAGE_KEY) || {},
+);
```

Comment:
Please use the `createStorageRef` and then `useStorageRef` composable helper instead.